File: myflix-frontend/src/app/model/movie-details-response.ts
export type MovieDetailsResponse = {
    id: number;
    addedAt: string;
    overview: string | null;
    title: string | null;
    posterPath: string | null;
    backdropPath: string | null;
    releaseDate: string | null;
    runtimeMinutes: number | null;
}

File: myflix-core/src/main/java/com/maszlovicskrisztian/myflix_core/dtos/response/MovieDetailsResponse.java
package com.maszlovicskrisztian.myflix_core.dtos.response;

import java.time.LocalDate;
import java.time.LocalDateTime;

public record MovieDetailsResponse(
        Long id,
        LocalDateTime addedAt,
        String overview,
        String title,
        String posterPath,
        String backdropPath,
        LocalDate releaseDate,
        Integer runtimeMinutes
) {
}
